app/views/observations/form/_projects.html.erb

```diff
@@ -22,6 +22,17 @@
     </div>
   <% end %>
 
+ <div class="mt-3 row" id="observation_projects">
+   <%= fields_for(:project) do |f_p| %>
+      <% if @suspect_checked_projects.any? %>
+        <%= check_box_with_label(
+          form: f_p, field: :ignore_proj_conflicts,
+          label: :form_observations_projects_ignore_project_constraints.t,
+          class: "alert alert-warning"
+        ) %>
+      <% end %>
+</div>
```

Comment:
This indentation confused me.  I created an alternative in https://github.com/MushroomObserver/mushroom-observer/pull/1700 that I find less confusing.  Not 100% it is exactly equivalent, but it seems to work after some light testing.